checks/raw/pinned_dependencies.go

```diff
@@ -187,10 +189,20 @@ func isDockerfile(pathfn string, content []byte) bool {
 }
 
 func collectDockerfilePinning(c *checker.CheckRequest, r *checker.PinningDependenciesData) error {
-	return fileparser.OnMatchingFileContentDo(c.RepoClient, fileparser.PathMatcher{
+	err := fileparser.OnMatchingFileContentDo(c.RepoClient, fileparser.PathMatcher{
 		Pattern:       "*Dockerfile*",
 		CaseSensitive: false,
 	}, validateDockerfilesPinning, r)
+	if err != nil {
+		return err
+	}
+	//nolint:errcheck
+	remediationMetadata, _ := remediation.New(c)
```

Comment:
now that we're in a function with an error return value, lets return it?

Reply:
Done in https://github.com/ossf/scorecard/pull/3632/commits/ae557a454330a8b10f878f835775a2af3bbb09fc